pylib/gyp/generator/gypd.py

```diff
@@ -81,7 +81,7 @@ def GenerateOutput(target_list, target_dicts, data, params):
         input_file_stem = input_file[:-4]
         output_file = input_file_stem + params["options"].suffix + ".gypd"
 
-        if not output_file in output_files:
+        if output_file not in output_files:
             output_files[output_file] = input_file
```

Comment:
```suggestion
        output_files[output_file] = output_files.get(output_file, input_file)
```
`dict.get()` can be used to simplify many of modifications made below.